xc-admin/packages/xc-admin-cli/src/index.ts

```diff
@@ -0,0 +1,90 @@
+import { Keypair, PublicKey } from "@solana/web3.js";
+import { program } from "commander";
+import { PythCluster } from "@pythnetwork/client/lib/cluster";
+import { getPythClusterApiUrl } from "@pythnetwork/client/lib/cluster";
+import { AnchorError, AnchorProvider, Program } from "@coral-xyz/anchor";
+import fs from "fs";
+import SquadsMesh from "@sqds/mesh";
+import NodeWallet from "@coral-xyz/anchor/dist/cjs/nodewallet";
+import { proposeInstructions } from "xc-admin-common";
+
+const PROGRAM_AUTHORITY_ESCROW = new PublicKey(
+  "escMHe7kSqPcDHx4HU44rAHhgdTLBZkUrU39aN8kMcL"
+);
+const BPF_UPGRADABLE_LOADER = new PublicKey(
+  "BPFLoaderUpgradeab1e11111111111111111111111"
+);
+program
+  .name("xc-admin-cli")
+  .description("CLI for interacting with Pyth's xc-admin")
+  .version("0.1.0");
+
+program
+  .command("accept-authority")
+  .description("Accept authority from the program authority escrow")
+  .requiredOption("-c, --cluster <network>", "solana cluster to use")
+  .requiredOption("-w, --wallet <filepath>", "path to the operations key")
+  .requiredOption(
+    "-p, --program-id <pubkey>",
+    "program whose authority we want to transfer"
+  )
+  .requiredOption(
+    "-a, --current <pubkey>",
+    "current authority (before the transfer)"
+  )
+  .requiredOption("-v, --vault <pubkey>", "multisig address")
```

Comment:
Not necessary as we are not gonna use it manually but i think hard-coding this value is easier (for testing, ...)